src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js

```diff
@@ -41,10 +42,38 @@ const DiagnosisAndTreatment = () => {
       };
     });
   };
-
   const treatmentDetails = t('Treatment details');
   const diagnosisDetails = t('Diagnosis details');
   const findingsDetails = t('Findings details');
+  React.useEffect(() => {
+    const { items } = questionnaireResponse;
+    if (items) {
+      items.forEach((item) => {
+        if (item.answer) {
+          switch (item.text) {
```

Comment:
the switch needs to be on the linkId, the text can be changed. it's better to put the text in a comment then the code will be more readable